HST/hst_helper/query_utils.py

```diff
@@ -0,0 +1,194 @@
+##########################################################################################
+# hst_helper/query_utils.py
+##########################################################################################
+import os
+import time
+
+import julian
+import pdslogger
+
+from astroquery.mast import Observations
+
+from . import (START_DATE,
+               END_DATE,
+               RETRY)
+from .fs_utils import (get_format_term,
+                       get_visit)
+from product_labels.suffix_info import (ACCEPTED_SUFFIXES,
+                                        ACCEPTED_LETTER_CODES,
+                                        INSTRUMENT_FROM_LETTER_CODE)
+
+def ymd_tuple_to_mjd(ymd):
+    """Return Modified Julian Date.
+    Input:
```

Comment:
In general these would be more readable if there were a blank line between the first line and the Input section